### PR TITLE
Center contents of tables on /edit/sessions-speakers

### LINKED ISSUE #372: Make the table of switches center aligned for mobile on /edit/sessions-speakers
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

image

**Expected behavior:**
image

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
The tables on /edit/sessions were not mobile friendly, this issue makes them responsive for mobile
#### Changes proposed in this pull request:

image

- tables are responsive now for mobiles
- the buttons have been centered for mobile 

Fixes #372

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@fossasia/open-event-frontend Please review 
http://cc96-opev-fe-omega.herokuapp.com/events/963d3ba5/edit/sessions-speakers

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -252,24 +264,36 @@
...
+                <td class="{{if device.isMobile 'center' 'right'}} aligned">
+                  <label class="{{if field.isFixed 'required'}}">
+                    {{field.name}}
+                  </label>
+                </td>
                 <td class="center aligned">
-                  <div class="ui slider checkbox">
-                    {{input type='checkbox' checked=field.isIncluded disabled=field.isFixed}}
-                  </div>
+                  {{ui-checkbox class='slider' checked=field.isIncluded disabled=field.isFixed label=(if device.isMobile (t 'Include'))}}
```
Comment (niranjan94) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: mutate `field.isIncluded` on change. This another thing that is required when `ui-checkbox` is used.

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -252,24 +264,36 @@
...
                 </td>
                 <td class="center aligned">
-                  <div class="ui slider checkbox">
-                    {{input type='checkbox' checked=field.isRequired disabled=field.isFixed}}
-                  </div>
+                  {{ui-checkbox class='slider' checked=field.isRequired disabled=field.isFixed label=(if device.isMobile (t 'Require'))}}
```
Comment (niranjan94) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: mutate `field. isRequired` on change. This another thing that is required when `ui-checkbox` is used.

**Comment by CosmicCoder96** [comment]
@niranjan94 I have made the changes, please review

**Pull request merged**